File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/CuentaSistemaModificarRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class CuentaSistemaModificarRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idUsuario;
	private String nombres;
	private String apePaterno;
	private String apeMaterno;
	private Date fecNacimiento;
	private String sexo;
	private Long idArea;
	private Long idCargo;
	private String usuario;
	private String contrasenia;
	private Long idPerfil;
	private Long idUsuarioModif;
	private List<Long> listaServicios;
	private Integer flgReporteDisociado;
}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/cuenta-sistema-modificar.request.ts
export class CuentaSistemaModificarRequest {
    idUsuario: number;
    nombres: string;
    apePaterno: string;
    apeMaterno: string;
    fecNacimiento: Date;
    sexo: string;
    idArea: number;
    idCargo: number;
    usuario: string;
    contrasenia: string;
    idPerfil: number;
    idUsuarioModif: number
    listaServicios: number[];
    flgReporteDisociado: number;
}